mods/ra/rules/infantry.yaml

```diff
@@ -114,6 +114,7 @@ E2:
 		DefaultAttackSequence: throw
 	Explodes:
 		Weapon: UnitExplodeSmall
+		EmptyWeapon: UnitExplodeSmall
```

Comment:
Please change this in a new commit (or squash the commits if you can do it) to `EmptyWeapon: UnitExplode` as pchote requested.